app/lib/utils/string.js

```diff
@@ -0,0 +1,3 @@
+export function escapeShellArgument (text) {
```

Comment:
Removed escapeShellArgument and instead added php-escape-shell module from npm :)